source/jormungandr/jormungandr/autocomplete/geocodejson.py

```diff
@@ -349,10 +350,47 @@ def _check_response(cls, response, uri):
             raise TechnicalError('error in autocomplete request')
 
     @classmethod
-    def response_marshaler(cls, response_bragi, uri=None):
+    def _clean_response(cls, response, depth=3):
+
+        def _clear_object(obj, dep):
+            if dep == -1:
```

Comment:
Since we are already checking the `depth` before calling this method we don't have to do it another time